File: benchmarks/py/sum_loop.py
import sys


def sum_loop(n: int) -> int:
  acc = 0
  i = 1
  while i <= n:
    acc += i
    i += 1
  return acc


if __name__ == "__main__":
  n = int(sys.argv[1]) if len(sys.argv) > 1 else 0
  print(sum_loop(n))

File: benchmarks/js/sum_loop.js
function sumLoop(n) {
  let acc = 0;
  for (let i = 1; i <= n; i++) {
    acc += i;
  }
  return acc;
}

const n = process.argv[2] ? parseInt(process.argv[2]) : 0;
console.log(sumLoop(n));
